Fix security/pcsc-lite's include path to match what pkg-config(1) says and
cmake uses.
https://github.com/open-eid/qt-common/pull/121

Index: common/QPCSC_p.h
--- common/QPCSC_p.h.orig
+++ common/QPCSC_p.h
@@ -30,8 +30,8 @@
 #include <winsock2.h>
 #include <winscard.h>
 #else
-#include <PCSC/wintypes.h>
-#include <PCSC/winscard.h>
+#include <wintypes.h>
+#include <winscard.h>
 #include <arpa/inet.h>
 #endif
 
